docker-compose.yml

```diff
@@ -4,24 +4,32 @@ volumes:
   repository-service-tuf-api-data:
   repository-service-tuf-redis-data:
   repository-service-tuf-pgsql-data:
-
 services:
   postgres:
     image: postgres:17.5-alpine3.21
     ports:
       - "5433:5432"
     environment:
-      - POSTGRES_PASSWORD=secret
+      - POSTGRES_PASSWORD_FILE=/run/secrets/postgres_password
```

Comment:
For the password, we keep it "readable" because this compose is used only for development/tests